js/apps/admin-ui/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "admin-ui",
```

Comment:
This package we want to publish as well

Reply:
It's not published right now though right? The publish configuration can be added when this will actually be prepped to publish.

Reply:
right we can change it after as well